### ISSUE TITLE
Private clone

### ISSUE THREAD

**Issue by Darklightsite** [VectorXz/alienworlds_acc_monitor#18]
Hy.
Can u help me create online clone to private usage? 
Your site load too long and have many error

**Comment by Darklightsite** [comment]
![Screenshot_2021-06-05-03-26-27-241_com android chrome](https://user-images.githubusercontent.com/84232138/120875777-e2cc8380-c5ad-11eb-8060-f047cc44823e.jpg)
Not load tlm

**Comment by VectorXz** [comment]
Currently, if you want to hosted on your own, you must also deploy backend server in folder "backend" that is using Express.JS too.
Then change link in frontend to use your api.

The error message is came from server was overloaded by a huge requests, and also wax API server overload, which I also look for a workaround to fix this.

**Comment by Darklightsite** [comment]
> Currently, if you want to hosted on your own, you must also deploy backend server in folder "backend" that is using Express.JS too.
> Then change link in frontend to use your api.
> 
> The error message is came from server was overloaded by a huge requests, and also wax API server overload, which I also look for a workaround to fix this.

I create vercel.app site, but loading tlm is too long :(

**Comment by VectorXz** [comment]
yep because it's still using our api server at api.alienworlds.fun.
You can look into the code.
That's why it's work the same way as you're using alienworlds.fun.

As I said, if you want to host on your own, you must also deploy backend server somewhere else because vercel does not support Express.js. Then change url in components to your own api server, that's all.
